examples/my_token.rs

```diff
@@ -0,0 +1,238 @@
+        .transfer(to, amount)
+        .send()
+        .await?
+        .await?
+        .expect("transfer tx returned non");
+    println!("transfer tx: {:?}", tx.transaction_hash);
+
+    let from_balance_after = my_token.balance_of(from).call().await?;
+    println!("from balance after : {}", from_balance_after);
+    let to_balance_after = my_token.balance_of(to).call().await?;
+    println!("to balance after : {}", to_balance_after);
+
+    Ok(())
+}
+
+async fn approve(
+    my_token_owner_signer: &MyTokenType,
+    owner: Address,
+    spender: Address,
+    amount: U256,
+) -> eyre::Result<()> {
+    println!("--- Approve");
+    let mut approved_amount: U256 = my_token_owner_signer.allowance(owner, spender).call().await?;
+    println!("approved amount before : {}", approved_amount);
+
+    let approve_tx: TransactionReceipt = my_token_owner_signer
+        .approve(spender, amount)
+        .send()
+        .await?
+        .await?
+        .expect("approve tx returned non");
+    println!("approve tx: {:?}", approve_tx.transaction_hash);
+
+    approved_amount = my_token_owner_signer.allowance(owner, spender).call().await?;
+    println!("approved amount after : {}", approved_amount);
+
+    Ok(())
+}
+
+async fn transfer_from(
+    my_token: &MyTokenType,
+    from: Address,
+    to: Address,
+    amount: U256,
+) -> eyre::Result<()> {
+    println!("--- Transfer from");
+    let from_balance_before: U256 = my_token.balance_of(from).call().await?;
+    println!("from balance before : {}", from_balance_before);
+    let to_balance_before: U256 = my_token.balance_of(to).call().await?;
+    println!("to balance before : {}", to_balance_before);
+
+    let tx: TransactionReceipt = my_token
+        .transfer_from(from, to, amount)
+        .send()
+        .await?
+        .await?
+        .expect("transfer from tx returned non");
+    println!("transfer from tx: {:?}", tx.transaction_hash);
+
+    let from_balance_after = my_token.balance_of(from).call().await?;
+    println!("from balance after : {}", from_balance_after);
+    let to_balance_after = my_token.balance_of(to).call().await?;
+    println!("to balance after : {}", to_balance_after);
+
+    Ok(())
+}
+
+async fn init_fixtures() -> eyre::Result<Fixtures>  {
+    dotenv().ok();
+
+    let program_address = std::env::var(MY_TOKEN_PROGRAM_ADDRESS)
+        .map_err(|_| eyre!("No {} env var set", MY_TOKEN_PROGRAM_ADDRESS))?;
+    let alice_key_path = std::env::var(ALICE_PRIV_KEY_PATH)
+        .map_err(|_| eyre!("No {} env var set", ALICE_PRIV_KEY_PATH))?;
+    let rpc_url = std::env::var(RPC_URL).map_err(|_| eyre!("No {} env var set", RPC_URL))?;
+    let bob_key_path = std::env::var(BOB_PRIV_KEY_PATH)
+        .map_err(|_| eyre!("No {} env var set", BOB_PRIV_KEY_PATH))?;
+
+    let provider = Provider::<Http>::try_from(rpc_url)?;
+    let my_token_address: Address = program_address.parse()?;
+
+    let alice_private_key = read_secret_from_file(&alice_key_path)?;
```

Comment:
Why not keep this in the `.env`?

Reply:
For the sake of this project we definitely can put that in `.env`
In other projects I was keeping the keys in the separate location to to accidentally copy & paste them when sharing contents of `.env` with other developers.

Reply:
Makes sense 👍🏻